UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Webjob.UnitTests/Services/FulfilmentAncillaryFilesTest.cs

```diff
@@ -0,0 +1,51 @@
+﻿using FakeItEasy;
+using Microsoft.Extensions.Options;
+using NUnit.Framework;
+using System.Threading.Tasks;
+using UKHO.ExchangeSetService.Common.Configuration;
+using UKHO.ExchangeSetService.Common.Helpers;
+using UKHO.ExchangeSetService.FulfilmentService.Services;
+
+namespace UKHO.ExchangeSetService.Webjob.UnitTests.Services
+{
+    [TestFixture]
+    public class FulfilmentAncillaryFilesTest
+    {
+        public IOptions<FileShareServiceConfiguration> fakefileShareServiceConfig;
+        public IFileSystemHelper fakeFileSystemHelper;
+        public FulfilmentAncillaryFiles fulfilmentAncillaryFiles;
+        public string batchId = "7b4cdf10-adfa-4ed6-b2fe-d1543d8b7272";
+        public string exchangeSetPath = string.Empty;
+
+        [SetUp]
+        public void Setup()
+        {
+            fakefileShareServiceConfig = Options.Create(new FileShareServiceConfiguration()
+            { SerialFileName = "TEST.ENC" });
+            fakeFileSystemHelper = A.Fake<IFileSystemHelper>();
+            fulfilmentAncillaryFiles = new FulfilmentAncillaryFiles(fakefileShareServiceConfig, fakeFileSystemHelper);
+        }
+
+        [Test]
+        public async Task WhenInvalidCreateSerialEncFileRequest_ThenReturnFalseResponse()
+        {
+            exchangeSetPath = "";
```

Comment:
Remove exchangeSetPath="" as at line 18 it is already assigned as empty. i.e. public string exchangeSetPath = string.Empty;

Reply:
Resolved